### PR TITLE
Include req's in setup.py

### LINKED ISSUE #150: plotly shouldn't be req'd
Only import plotly (in experiments) if plotting is specified.

### PR DESCRIPTION
**Pull request by BoltzmannBrain**
fixes #150

### CODE REVIEW

**Comment by BoltzmannBrain** [comment]
@scottpurdy, @rhyolight, or @lscheinkman please review.

@lscheinkman this should address the issues you had installing fluent.

File: README.md [hunk 1]
```diff
@@ -21,7 +21,9 @@ The motivation here is we would like to move quickly in research while maintaini
 Requirements:
 
 - [NuPIC](https://github.com/numenta/nupic)
+- [NuPIC.research/classification](https://github.com/numenta/nupic.research/tree/master/classification)
```
Comment (lscheinkman) [inline README.md]: @BoltzmannBrain This repo does not have a `setup.py`. Should I just add `nupic.research/classification` to my `PYTHON_PATH` instead?

Comment (BoltzmannBrain) [inline README.md]: yes, I'll specify this in the readme

File: README.md [hunk 2]
```diff
@@ -29,10 +31,14 @@ To install, run:
...
     python setup.py install
 
-Then, set up the following environment variables with your REST API credentials:
+Then, set up the following environment variables with your Cortical.io REST API credentials:
 
     export CORTICAL_API_KEY=api_key
 
+Add nupic.research/classification to your Python path:
+
+	export PYTHONPATH=$PYTHONPATH:/<path_to_nupic.research/classification>
```
Comment (lscheinkman) [inline README.md]: @BoltzmannBrain It looks like the correct path is `/<path_to_nupic.research>` without the `classification` part.
See https://github.com/numenta/nupic.fluent/blob/master/fluent/models/classify_htm.py#L24

**Pull request merged**